hardhat.config.js

```diff
@@ -108,10 +108,9 @@ module.exports = {
     // apiKey: secrets.ETHERSCAN_API,
     apiKey: {
       mainnet: process.env.ETHERSCAN_API,
-      goerli: process.env.ETHERSCAN_API
+      goerli: process.env.ETHERSCAN_API,
```

Comment:
lets remove any files from the commit were not making many changes in. itll help me avoid merge conflicts later

Reply:
seems like my prettier config changed this. let me try to revert this